IPython/utils/py3compat.py

```diff
@@ -172,9 +172,6 @@ def isidentifier(s, dotted=False):
             return all(isidentifier(a) for a in s.split("."))
         return s.isidentifier()
 
-    xrange = range
-    def iteritems(d): return iter(d.items())
-    def itervalues(d): return iter(d.values())
```

Comment:
Be careful removing stuff, even if it's not used anywhere in IPython; we've had a couple of times before when we've removed things from IPython and then found that we're still using them in another module, like ipykernel.